src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -42,6 +43,8 @@ def notifiable_link_text
       repository = @notification.event_payload['repository']
       arch = @notification.event_payload['arch']
       "Package #{package} on #{project} project failed to build against #{repository} / #{arch}"
+    when 'Event::CreateReport'
+      "Report for a #{@notification.event_payload['reportable_type'].downcase} created"
```

Comment:
You are missing `Report.reason` in here as this is the only place we can show this right now.

Reply:
done